CONTRIBUTING.md

```diff
@@ -1,17 +1,45 @@
-# Contributing to RocksDB
+# Contributing to ForSt
 
 ## Code of Conduct
 The code of conduct is described in [`CODE_OF_CONDUCT.md`](CODE_OF_CONDUCT.md)
 
-## Contributor License Agreement ("CLA")
+## Basic Development Workflow
+As most open-source projects in github, ForSt contributors work on their fork, and send pull requests to ForSt’s repo. After a reviewer approves the pull request and all the CI check passed, a ForSt team member will merge it.
```

Comment:
their fork -> their forks?
all the CI check passed-> all the CI checks are passed?